app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
```

Comment:
Was this something generated for you? Are the empty methods doing something devise-y here?